src/lib.rs

```diff
@@ -572,6 +546,33 @@ impl TestKit {
         self.poll_events();
     }
 
+    // Try to modify the test network configuration.
+    fn do_config_update(&mut self) {
```

Comment:
Nitpick: imo, `update_config` sounds better. Also, why "try" in the comment? As far as I see, there is no `Result` returned or possibility to `panic` here. I'd rephrase
> Update test network configuration if such an update has been scheduled with `propose_configuration_change()`.

Reply:
Committed